index.template.html

```diff
@@ -77,18 +77,23 @@
         <!-- // NAVBAR -->
 
         <!-- // QR MODAL -->
-        <div class="modal fade" id="qrModal" tabindex="-1" role="dialog" aria-labelledby="qrModalLabel" aria-hidden="true">
+        <div class="modal fade" style="width: 375px;" id="qrModal" tabindex="-1" role="dialog" aria-labelledby="qrModalLabel" aria-hidden="true">
           <div class="modal-dialog modal-dialog-centered" role="document">
             <div class="modal-content exportKeysModalColor">
               <div class="modal-header">
-                <h5 class="modal-title" id="qrModalLabel"><span data-i18n="address" >Address</span> QR</h5>
+                <h5 class="modal-title" data-i18n="receive" id="qrModalLabel">Receive</h5>
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                   <i class="fa-solid fa-xmark closeCross"></i>
                 </button>
               </div>
               <div class="modal-body center-text">
                 <code id="ModalQRLabel" class="wallet-code"></code>
                 <div id="ModalQR" style="padding: 17px 10px 10px 10px;" class="auto-fit"></div>
+                <center>
+                  <div id="ModalQRReceiveTypeBtn" onclick="MPW.guiToggleReceiveType()" style="cursor: pointer; border: 0px; border-radius: 7px; padding: 6px 10px; margin: 0px 1px; background: linear-gradient(183deg, #9621ff9c, #7d21ffc7); color: #fff; font-weight: bold; width: fit-content; margin: 20px 0px;">
+                    Change to Address
```

Comment:
no i18n on this one? `Change to Address`

Reply:
That HTML should be rendered by contacts-book.js afaik (If not, I'll fix that and render it by JS, since we can't use Static rendering here).